app/collections/settings.cjsx

```diff
@@ -63,47 +64,50 @@ module.exports = React.createClass
   redirect: ->
     @transitionTo 'collections'
 
-  checkUserRole: ->
-    if @props.user?
-      apiClient.type('collection_roles').get(collection_id: @props.collection.id, user_id: @props.user.id)
-        .catch ->
-          []
-        .then ([roles]) ->
-          roles?
-
   confirmDelete: ->
     alert (resolve) =>
       <div className="confirm-delete-dialog content-container">
         <CollectionDeleteDialog {...@props} onComplete={resolve} />
       </div>
 
+  public_collection: ->
```

Comment:
yeah for sure, i'm always changing this after i see them...default style for me.